ipn/ipn_view_helpers.go

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Tailscale Inc & AUTHORS
```

Comment:
instead of filename `ipn_view_helpers.go`, let's just add it to the bottom of ipn/serve.go.